# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

uint64 uid
string type
geometry_msgs/Pose2D pose
float32 width
float32 length
geometry_msgs/Pose2D parking_take_off
bool one_time
